aten/src/ATen/templates/StorageDerived.cpp

```diff
@@ -10,19 +10,20 @@
 
 namespace at {
 
-${Storage}::${Storage}(Context* context):
-    Storage(${THStorage}_new(${state})), context(context) {}
+${Storage}::${Storage}():
+    Storage(${THStorage}_new(${state})) {}
 
-${Storage}::${Storage}(Context* context, THStorage* storage):
-    Storage(storage), context(context) {}
+${Storage}::${Storage}(THStorage* storage):
+    Storage(storage) {}
 
-${Storage}::${Storage}(Context* context, size_t storage_size)
-  : Storage(${THStorage}_newWithSize(${state,} storage_size)), context(context) {}
+${Storage}::${Storage}(size_t storage_size)
+  : Storage(${THStorage}_newWithSize(${state,} storage_size)) {}
 
-${Storage}::${Storage}(Context* context, size_t size, Allocator* allocator)
-  : Storage(nullptr), context(context) {
-  storage = ${THStorage}_newWithAllocator(${state,} size, allocator);
-  ${THStorage}_clearFlag(${state,} storage, TH_STORAGE_RESIZABLE);
+${Storage}::${Storage}(size_t size, Allocator* allocator)
+  : Storage(nullptr) {
+//  storage = ${THStorage}_newWithAllocator(${state,} size, allocator);
+  storage = new THStorage(ScalarType::${ScalarName}, size, allocator, TH_STORAGE_RESIZABLE);
+  THStorage_setResizable(storage, TH_STORAGE_RESIZABLE);
```

Comment:
Don't use this constant anymore, right? Also, don't you want to say this is NOT resizable.